x-pack/legacy/plugins/triggers_actions_ui/np_ready/public/application/sections/alerts_list/components/alerts_list.tsx

```diff
@@ -0,0 +1,297 @@
+        title: i18n.translate(
+          'xpack.triggersActionsUI.sections.alertsList.unableToLoadAlertsMessage',
+          {
+            defaultMessage: 'Unable to load alerts',
+          }
+        ),
+      });
+    } finally {
+      setIsLoadingAlerts(false);
+    }
+  }
+
+  const alertsTableColumns = [
+    {
+      field: 'name',
+      name: i18n.translate(
+        'xpack.triggersActionsUI.sections.alertsList.alertsListTable.columns.nameTitle',
+        { defaultMessage: 'Name' }
+      ),
+      sortable: false,
+      truncateText: true,
+      'data-test-subj': 'alertsTableCell-name',
+    },
+    {
+      field: 'tagsText',
+      name: i18n.translate(
+        'xpack.triggersActionsUI.sections.alertsList.alertsListTable.columns.tagsText',
+        { defaultMessage: 'Tags' }
+      ),
+      sortable: false,
+      'data-test-subj': 'alertsTableCell-tagsText',
+    },
+    {
+      field: 'alertType',
+      name: i18n.translate(
+        'xpack.triggersActionsUI.sections.alertsList.alertsListTable.columns.alertTypeTitle',
+        { defaultMessage: 'Type' }
+      ),
+      sortable: false,
+      truncateText: true,
+      'data-test-subj': 'alertsTableCell-alertType',
+    },
+    {
+      field: 'schedule.interval',
+      name: i18n.translate(
+        'xpack.triggersActionsUI.sections.alertsList.alertsListTable.columns.intervalTitle',
+        { defaultMessage: 'Runs every' }
+      ),
+      sortable: false,
+      truncateText: false,
+      'data-test-subj': 'alertsTableCell-interval',
+    },
+    {
+      name: '',
+      width: '40px',
+      render(item: AlertTableItem) {
+        return (
+          <CollapsedItemActions key={item.id} item={item} onAlertChanged={() => loadAlertsData()} />
+        );
+      },
+    },
+  ];
+
+  const toolsRight = [
+    <TypeFilter
+      key="type-filter"
+      onChange={(types: string[]) => setTypesFilter(types)}
+      options={Object.values(alertTypesIndex || {})
+        .map(alertType => ({
+          value: alertType.id,
+          name: alertType.name,
+        }))
+        .sort((a, b) => a.name.localeCompare(b.name))}
+    />,
+    <ActionTypeFilter
+      actionTypes={actionTypes}
+      onChange={(ids: string[]) => setActionTypesFilter(ids)}
+    />,
+  ];
+
+  if (canSave && createAlertUiEnabled) {
+    toolsRight.push(
+      <EuiButton
+        key="create-alert"
+        data-test-subj="createAlertButton"
+        fill
+        iconType="plusInCircle"
+        iconSide="left"
+        onClick={() => setAlertFlyoutVisibility(true)}
+      >
+        <FormattedMessage
+          id="xpack.triggersActionsUI.sections.alertsList.addActionButtonLabel"
+          defaultMessage="Create"
+        />
+      </EuiButton>
+    );
+  }
+
+  return (
+    <section data-test-subj="alertsList">
+      <Fragment>
+        <EuiSpacer size="m" />
+        <AlertsContext.Provider value={{ alertFlyoutVisible, setAlertFlyoutVisibility }}>
+          <EuiSearchBar
```

Comment:
I don't think you actually need an `EuiSearchBar` here since you're only using the query text. `EuiFieldText` has `append` and `prepend` props you could use to place the bulk actions and filter buttons inline with it. `EuiSearchBar` is a super complex component and I'd avoid using it unless you really need to.

Reply:
That makes sense, I went ahead and made the change here: https://github.com/elastic/kibana/pull/48959/commits/0b48b654a320f0143903e5a7bd9c2c3cc9e40b57.